.editorconfig

```diff
@@ -0,0 +1,23 @@
+# The .editorconfig is used to maintain consistent code style.
+# The .editorconfig file is supported by most text editors.
+# See https://editorconfig.org
+
+root = true
+
+[*]
+insert_final_newline = true
```

Comment:
Let's use `insert_final_newline` so when we `cat` files, the output looks like:
```
> cat my-file.txt
contents
of
file
> 
```
instead of
```
> cat my-file.txt
contents
of
file > 
```